RELEASE.md

```diff
@@ -18,10 +18,10 @@ All the Python packaging instructions in the `pyproject.toml` file to wrap your
 > [!IMPORTANT]
 > There is no need to update the version in `pyproject.toml` as it is automatically updated by `hatch` when generating the package.
 
-3. Optional - if there is a newer release of `conda-store-ui`, update the `conda-store-ui` dependency in the `package.json` file.
+3. Optional - if there is a newer release of `conda-store-ui`, update the `conda-store-ui` dependency in the `package.json` file. Then run an install
```

Comment:
```suggestion
3. Optional - if there is a newer release of `conda-store-ui`, update the `conda-store-ui` dependency in the `package.json` file. Then run and install
```

Also, for yarn v3 (berry) you could use `yarn up @conda-store/conda-store-ui@<version>`

Reply:
oh that's a good hint! I'll update the docs with this instead.